pkg/netceptor/addr.go

```diff
@@ -4,17 +4,17 @@ import "fmt"
 
 // Addr represents an endpoint address on the Netceptor network.
 type Addr struct {
-	network string
-	node    string
-	service string
+	NetworkStr string
```

Comment:
Great work on this, can you quickly run through why the vars need to be exported please?

Reply:
The vars need to be exported in order for netceptor_test to create an Addr with set vars for this test.